### PR TITLE
Update README.md - last Win7 version

### PR DESCRIPTION
**Pull request by PizzaProgram**
- add last working Win7 link: v1.20
- some other small fixes, like better listing, formatting...

### CODE REVIEW

**Comment by PizzaProgram** [comment]
@rumanzo Thank you very much for this great work!

You may do not know about it, but latest GO dropped Win7 support.
Everything compiled with that, breaks on Win7 (both 32 & 64 bit).

Here is what happens:
```
C:\Users\IT>E:\bt2qbt_v1.21_amd64.exe
Exception 0xc0000005 0x8 0x0 0x0
PC=0x0

runtime.asmstdcall()
        /usr/local/go/src/runtime/sys_windows_amd64.s:65 +0x75 fp=0x22fca0 sp=0x22fc80 pc=0x4626d5
rax     0x0
rbx     0x666e00
rcx     0x6b9900
rdi     0x7fffffde000
rsi     0x22fea0
rbp     0x22fde0
rsp     0x22fc78
r8      0x0
r9      0x22fee0
r10     0x68b738
r11     0x21
r12     0x22fec0
r13     0x1
r14     0x6667a0
r15     0x0
rip     0x0
rflags  0x10293
cs      0x33
fs      0x53
gs      0x2b
```

If you do not wish to change GO version **back** to the last known Win7-compatible one,  
I'd like to recommend to apply a patch, where Your APP first checks the OS version minimum, 
and report it to the user, to download the last known working version: 1.20. (+link)

**Comment by rumanzo** [comment]
Hello! Thank you for you notice, I didn't think about it. 
I suggest the better way is to resolv https://github.com/rumanzo/bt2qbt/issues/47 and build new version with to the last known Win7-compatible version of go. I think it is not a problem, I don't use newest features in my code.

File: README.md [hunk 1]
```diff
@@ -60,7 +64,7 @@ Help (from cmd or powershell)
 
 ```
 Usage:
-  bt2qbt_v1.999_amd64.exe [OPTIONS]
+  bt2qbt_v1.99_amd64.exe [OPTIONS]
```
Comment (rumanzo) [inline README.md:67]: Why? :-)

Comment (PizzaProgram) [reply README.md:67]: All your releases are so far x.yy digit long.
https://github.com/rumanzo/bt2qbt/releases

Comment (PizzaProgram) [reply README.md:67]: I was thinking about this for at least 15 minutes long, if I should change or not.
My other idea was:
`bt2qbt_v.........._amd64.exe`

Comment (rumanzo) [reply README.md:67]: It's doesn't matter actually, just it should be not equal actual releases for avoid copy paste mistakes.
Just asking)

**Comment by PizzaProgram** [comment]
> I suggest the better way is to resolv https://github.com/rumanzo/bt2qbt/issues/47 and build new version with to the last known Win7-compatible version of go. I think it is not a problem, I don't use newest features in my code.

I agree.
keeping Win7 compatibility would be the best solution :+1: 
There are still many PCs all over the word, those can be used perfectly fine as a torrenct PC running a simple Win7 behind a NAT router.
_(In my experience W10 64bit is running the same programs 4x - 8x slower because of "sandboxing".)_

File: README.md [hunk 2]
```diff
@@ -4,7 +4,8 @@
 
 bt2qbt is cli tool for export from uTorrent\Bittorrent into qBittorrent (convert)
 > [!IMPORTANT]
-> Actual version tested with uTorrent 3.5.5 (build 46206) and qBittorrent 4.4.2. It should work with older version utorrent and newer version of qBittorrent, but it isn't tested.
+> Actual version tested with uTorrent 3.5.5 (build 46206) and qBittorrent 4.4.2. It should work with older version utorrent and newer version of qBittorrent, but it isn't tested.  
+> _Last known working version for_ **Windows 7** is: [1.20](https://github.com/rumanzo/bt2qbt/releases/tag/v1.20)
```
Comment (rumanzo) [inline README.md:8]: Please remove this line. New version that coming soon will be win 7 compatible.

**Pull request merged**